torch/_functorch/autograd_function.py

```diff
@@ -171,8 +188,8 @@ def mark_dirty_error(*args, **kwargs):
 #         return x.exp()
 #
 #     @staticmethod
-#     def setup_context(ctx, outputs, x):
-#         y = outputs
+#     def setup_context(ctx, inputs, output):
```

Comment:
Did you actually swap the order? That wasn't reflected in the tests.

Reply:
This is just an outdated comment, this is now the correct order

Reply:
(see the python bindings)